miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -1,30 +1,67 @@
 package com.rakuten.tech.mobile.miniapp.js
 
 import android.webkit.JavascriptInterface
+import androidx.annotation.VisibleForTesting
 import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
 import com.rakuten.tech.mobile.miniapp.display.WebViewListener
 
+@Suppress("TooGenericExceptionCaught", "SwallowedException")
 /** Bridge interface for communicating with mini app. **/
 abstract class MiniAppMessageBridge {
     private lateinit var webViewListener: WebViewListener
 
     /** Get provided id of mini app for any purpose. **/
     abstract fun getUniqueId(): String
 
+    /** Post permission request from external. **/
+    abstract fun requestPermission(
```

Comment:
Oh, sorry, I just noticed that the iOS side is using `requestPermission(permissionType: MiniAppPermissionType, completionHandler: @escaping (Result<String, Error>) -> Void)`. 

So maybe it makes more sense to move the `miniAppPermissionType` as the first parameter and `callback` as the second. What do you think?

Reply:
Just that all of the SDK has `suspend` and this would be the first place to have a callback.

Reply:
@ts-nitish-a-jain But the host app has to rely on `onRequestPermissionsResult` in the Activity, so it seems like using a `suspend` function would make things more complicated for them. I can't really think of a good way to make it work with `suspend`. How about you @minh-rakuten ?

Reply:
Okay. Makes sense. Thanks. I don't have any solution either for this at this moment.